src/families/tron/test-dataset.js

```diff
@@ -27,6 +27,7 @@ const dataset: DatasetTest<Transaction> = {
   currencies: {
     tron: {
       FIXME_ignoreAccountFields: [
+        "tronResources.bandwidth",
```

Comment:
can you comment **in the code** why we need to ignore these? it's not clear to me

Reply:
Because this account was not totally frozen but it should be the case right now.
I will delete this ignored fields and regenerate snapshots